common/src/objects/workload_state.rs

```diff
@@ -55,14 +59,13 @@ impl Display for PendingSubstate {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub enum RunningSubstate {
     Ok = 0,
-    DeleteFailed = 8,
 }
 
 impl From<i32> for RunningSubstate {
     fn from(x: i32) -> Self {
         match x {
             x if x == RunningSubstate::Ok as i32 => RunningSubstate::Ok,
-            _ => RunningSubstate::DeleteFailed,
+            _ => RunningSubstate::Ok,
```

Comment:
Do we need the 'match' at all, as in any case the return value is 'RunningSubstate::Ok'?